Bottom: cb8a166a49d92dc3423d11cc9eb021147c9b424a
Top:    e699287bf91c1e089e5380a9afd365f03b7906bb
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-31 16:11:41 -0700

Refresh of addrinfo-af-ib

---

diff --git a/include/rdma/rdma_cma.h b/include/rdma/rdma_cma.h
index 07fc747..4c4a057 100755
--- a/include/rdma/rdma_cma.h
+++ b/include/rdma/rdma_cma.h
@@ -77,6 +77,7 @@ enum rdma_port_space {
 #define RDMA_IB_IP_PORT_MASK 0x000000000000FFFFULL
 #define RDMA_IB_IP_PS_TCP    0x0000000001060000ULL
 #define RDMA_IB_IP_PS_UDP    0x0000000001110000ULL
+#define RDMA_IB_PS_IB        0x00000000013F0000ULL
 
 /*
  * Global qkey value for UDP QPs and multicast groups created via the
